study/cyber-tutor/index.html

```diff
@@ -0,0 +1,91 @@
+<html>
+    <head>
+        <title>Cyber Tutor</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+        <link rel="stylesheet" href="../../css/bootstrap4.5.0.min.css">
+        <link rel="stylesheet" href="../codemirror/codemirror.css">
+        <link rel="stylesheet" href="css/style.css">
+
+        <script src="https://kit.fontawesome.com/b3cde37b04.js" crossorigin="anonymous"></script>
+    </head>
+
+    <body>
+        <nav class="navbar navbar-expand-md bg-secondary navbar-dark mb-3">
+            <a class="navbar-brand" href="../../">TeCanal Kids Corner</a>
+
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#collapsibleNavbar">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+
+            <div class="collapse navbar-collapse" id="collapsibleNavbar">
+                <ul class="navbar-nav">
+                    <li class="nav-item">
+                        <a class="nav-link" href="../../play">
+                            <i class="fa fa-gamepad"></i>
+                            Play
+                        </a>
+                    </li>
+
+                    <li class="nav-item active">
+                        <i class="fa fa-paperclip"></i>
+                        Study
+                    </li>
+
+                    <li class="nav-item">
+                        <a class="nav-link" href="../../read">
+                            <i class="fa fa-book"></i>
+                            Read
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+
+        <section>
+            <div id="notification" style="display: none;"></div>
+
+            <div class="container">
+                <div style="text-align: center;">
+                    <h1 style="margin-bottom: 0;">Cyber Tutor</h1>
+                </div>
+				<br>
+				<table width="500" cellpadding = "5">
+				  <tr> 
+				  
+					<th>Term</th>
+					<th>Definition</th>
+				  </tr>
+				  <tr>
+					<td>Cybersecurity   </td>
+					<td>   Protection of information systems, programs, and data from unauthorized access, modification, and destruction</td>
+				  </tr>
+				  <tr>
+					<td>Hacker   </td>
+					<td>   Someone who uses a computer to get unauthorized access to an information system, program, or data</td>
+				  </tr>
+				</table>
+				<h3>
+                        <a href="quiz/">Start Review</a>
```

Comment:
Make this a [Bootstrap Button](https://getbootstrap.com/docs/4.0/components/buttons/).